lib/puppet/cloudpack/scripts/puppet-community.erb

```diff
@@ -43,7 +43,7 @@ function rpm_install() {
 
 function apt_install() {
   # Download and install the puppetlabs apt public
-  apt-key adv --recv-key --keyserver pool.sks-keyservers.net 4BD6EC30
+  apt-key adv --recv-key --keyserver pool.sks-keyservers.net 7F438280EF8D349F
```

Comment:
This keyserver seems to be toast afaict. Might be worth replacing with `pgp.mit.edu`?